### ISSUE TITLE
Grouped Windows List not grouping JetBrains apps

### ISSUE THREAD

**Issue by graineri** [linuxmint/cinnamon#10184]
Linux Mint 20.1
Cinnamon 4.8.6

This happens for GoLand, IntelliJ and others.
![image](https://user-images.githubusercontent.com/762503/123647516-85b79c80-d7fe-11eb-9782-7ea3d52b8631.png)

```
cat ~/.xsession-errors | grep grouped
Cjs-Message: 15:10:10.539: JS LOG: [LookingGlass/info] Loaded applet grouped-window-list@cinnamon.org in 36 ms
Cjs-Message: 09:35:09.962: JS LOG: [LookingGlass/info] Loaded applet grouped-window-list@cinnamon.org in 94 ms
```

**Comment by icarter09** [comment]
@graineri I was able to get the GWL to work correct for IntelliJ when I downloaded IntelliJ via the Software Manager instead of trying to use the version that I downloaded from the JetBrians.

![test_one](https://user-images.githubusercontent.com/4397312/124532284-3c5ae480-ddde-11eb-8061-a1badbe93459.png)

**Comment by graineri** [comment]
Thanks for your response. 

It seems those are Flatpak versions and are not always up-to-date.

**Comment by icarter09** [comment]
Yes I have noticed that as well with other frameworks not being up-to-date via the Software Manager. This should help to point to the issue with GWL and why it's not picking up on JetBrains products correctly.
